uktrade/data-hub-helpcentre
# Feat:Expose pingdom healthcheck endpoint

### Description
This change updates existing healthcheck endpoing (/check) to use the custom pingdom HTML format.

### How to test
Run the app locally
Access the healthcheck by going to http://localhost:8000/pingdom/ping.xml

You should see an ok status given the database is available and there is a default site.
